browser/src/Services/Menu/Menu.less

```diff
@@ -64,7 +64,7 @@
 
                 .highlight {
                     font-weight: bold;
-                    text-decoration: underline;
+                    color: #608be1;
```

Comment:
It would be cool to have this pulled from the current theme - could we move the `.highlight` style to a `styled-component` and use a theme color?

We could potentially add a `& .highlight { ... }` style here: https://github.com/onivim/oni/blob/80e16ee3f09d5173da7f3d181ca8f92389e98102/browser/src/Services/Menu/MenuComponent.tsx#L205

And we could grab some theme color - either a new one, or reuse one, like:
```
    & .highlight {
        font-weight: bold;
        color: ${p => p.theme["highlight.mode.normal.background"]};
    }
```

Reply:
I really like that way, it's cleaner thanks!